README.md

```diff
@@ -2,3 +2,50 @@
 
 This repository contains the Radio Frequency Front End (RFFE) EPICS Input/Output
 Controller (IOC) for LNLS Beam Position Monitor (BPM).
+
+## Building the Docker image
+
+In order to build the Docker image, you shall execute the following command
+
+```bash
+docker build \
+  -t ghcr.io/lnls-dig/rffe-epics-ioc \
+  -f docker/Dockerfile \
+  --build-arg JOBS=$(nproc) \
+  $(pwd)
+```
+
+The `JOBS` argument specifies the number of jobs (1 by default) to use `make`
+the application (EPICS base, its modules and the IOC itself). It will not
+change the resulting image, only the time for building the image.
+
+## Running the IOC
+
+Afterwards, you should create a container specifying the crate and BPM slot
+numbers, RFFEs' base IP address, and set up the network to inherent the host's
+configuration. For instance,
+
+```bash
+docker run \
+  -d \
+  --network host \
+  --mount type=bind,source=/opt/autosave,target=/opt/autosave \
```

Comment:
https://refspecs.linuxfoundation.org/FHS_3.0/fhs/ch05s12.html

Makes me think we should use `/var/opt`.

I had already wanted to use `/var`, but found a good directory for it inside of `/var`

Reply:
I agree. Just to be sure... When you say about using it, do you mean both in the container and host machine?

Reply:
Yups.

Reply:
Done. I used `/var/opt/rffe-epics-ioc/autosave`. Don't know if that will be our standard, but it at least matches the IOC path (`/opt/rffe-epics-ioc`) for now.

Reply:
I like it :)